R/comp_corr.R

```diff
@@ -1,12 +1,13 @@
 #' Computing corr
 #'
-#' This function computes the weighted correlation with penalty (no penalty if not lagged) after the lags are applied
-#' to the dataset and timepoints using the functions prep.data() and best.lag() function.
+#' This function computes the weighted correlation with a penalty for lags.
+#' It should only be used after the fixed lags have already been applied to the dataset and timepoints using the functions
+#' prep.data() and best.lag().
 #'
 #'
 #' @param data a lagged matrix with rows representing genes and columns representing
 #' different timepoints (NA's added when lags are needed)
-#' @param time a lagged matrix which contains all the timepoints represented in the data, NA is introduced when it is lagged
+#' @param time a lagged matrix with row representing each genes timepoint and column representing the length of timepoint, NA is introduced when it is lagged
```

Comment:
`rows representing each gene's` and `columns`.  Is it `length of timepoint` or `number of timepoints`?

Reply:
number of timepoints, I will fix this

Reply:
fixed